root/release/edit/information.tt

```diff
@@ -314,110 +314,253 @@
+              January 1st is not often the actual release date.
+              <strong>Please double-check that you have entered the release date correctly.</strong>') %]
+      </p>
+    <!-- /ko -->
+  </div>
+
+  <div class="bubble" data-bind="bubble: $root.countryBubble">
+    <p>
+      [% l('The {doc|country} describes where the release was made available.',
+           { doc => { href => doc_link('Release/Country'), target => '_blank' } }) %]
+    </p>
     <p>
-      [% l('The script attribute should be set to the script used for the release title and track titles.') %]
+      If you aren’t sure where it was released, please leave this unset.
     </p>
   </div>
 
+  <div class="bubble" data-bind="bubble: $root.catalogNumberBubble">
+    <!-- ko ifnot: target() && catNoLooksLikeASIN(target().catalogNumber()) -->
+      <p>
+        [% l('The {doc|catalog number} is assigned by the label and can often be found on the spine or near the barcode on physical media.',
+             { doc => { href => doc_link('Release/Catalog_Number'), target => '_blank' } }) %]
+      </p>
+      <p>
+        If you can’t find a catalog number, please leave this blank.
+      </p>
+    <!-- /ko -->
+    <!-- ko if: target() && catNoLooksLikeASIN(target().catalogNumber()) -->
+      <p>
+        [% l('The catalog number you have entered looks like an Amazon ASIN.') %]
+      </p>
+      <p>
+        [% l('If this is an Amazon ASIN, please do not enter it in the catalog number field, but add it as a URL relationship later.') %]
+      </p>
+    <!-- /ko -->
+  </div>
+
+  <div class="bubble" data-bind="bubble: $root.labelBubble">
+    <!-- ko with: target() && target().label() -->
+      <!-- ko if: $data.gid -->
+        <p data-bind="html: $data.selectionMessage()"></p>
+      <!-- /ko -->
+      <!-- ko ifnot: $data.gid -->
+        <p>
+          [% l('The {doc|label} is the imprint associated with the release.
+                Please also see the guidelines about {style|special purpose labels}.',
+               { doc => { href => doc_link('Label'), target => '_blank' },
+                 style => { href => doc_link('Style/Unknown_and_untitled/Special_purpose_label'), target => '_blank' } }) %]
+        </p>
+      <!-- /ko -->
+    <!-- /ko -->
+  </div>
+
   <div class="bubble" data-bind="bubble: $root.barcodeBubble">
     <p>
-      [% l('Please enter the barcode of the release you are entering, see <a href="{url}" target="_blank">Barcode</a> for more information.',
-           { url => doc_link('Barcode') }) %]
+      [% l('The {doc|barcode} is the numeric identifier (such as UPC or EAN) associated with the release.',
+           { doc => { href => doc_link('Barcode'), target => '_blank' } }) %]
     </p>
     <p>
-      [% l('If you do not know whether the release has a barcode or not, just leave this blank.') %]
+      [% l('If you don’t know whether the release has a barcode or not, please leave this blank.') %]
```

Comment:
Should we have a bit more depth in here, such as: "For example, most streaming releases do have a barcode, even though it's often not clearly displayed to the user."?

Also, should we specify "If you are certain the release has no barcode, check the relevant checkbox." or is that obvious enough?

Reply:
I think the checkbox is pretty straightforward. :-)

Regarding "most streaming releases do have a barcode"... hmm, my hesitation with that is that I'm not really sure what editors can do with that information. Leaving it blank seems like the only action one can take unless they're using a seeder that pulls the barcode out of the API or page source, in which case they're probably just going to leave this field alone.

Reply:
My hope with that would be that they would *not* assume there's no barcode and mark the box for that kind of release :) But maybe some other wording would work better.

Reply:
Maybe:

The {doc|catalog number} is assigned by the label and is often found on the spine or near the barcode on physical media.
Do not mark digital media releases as having no barcode unless you are familiar with the platform. Most mainstream digital platforms use barcodes, even when they are not shown.
If you can’t find a catalog number, please leave this blank.